packages/react/src/utils/SigCheck.ts

```diff
@@ -0,0 +1,28 @@
+import * as Lib from 'dm3-lib';
+import { stringify } from 'safe-stable-stringify';
+
+export async function checkSignature(
+    message: Lib.messaging.Message,
+    publicSigningKey: string,
+    accountAddress: string,
+    signature: string,
+): Promise<boolean> {
+    const isValid = await Lib.crypto.checkSignature(
+        publicSigningKey,
+        stringify(message)!,
+        signature,
+    );
+
+    if (!isValid) {
```

Comment:
Why not return false here already? After this there is no way the function can return true anymore.

Reply:
Yes, I simplified the function